patches/server/0954-added-missing-check.patch

```diff
@@ -0,0 +1,28 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: NonSwag <mrminecraft00@gmail.com>
+Date: Thu, 8 Dec 2022 20:25:05 +0100
+Subject: [PATCH] added missing check
+
+
+diff --git a/src/main/java/net/minecraft/server/network/ServerGamePacketListenerImpl.java b/src/main/java/net/minecraft/server/network/ServerGamePacketListenerImpl.java
+index d31a345edfffe39f127073fc3aec8b3489bae79c..5cd59ca136a8dd74b3c2bde0476b9630c3a8fd11 100644
+--- a/src/main/java/net/minecraft/server/network/ServerGamePacketListenerImpl.java
++++ b/src/main/java/net/minecraft/server/network/ServerGamePacketListenerImpl.java
+@@ -243,6 +243,7 @@ import org.bukkit.inventory.CraftingInventory;
+ import org.bukkit.inventory.EquipmentSlot;
+ import org.bukkit.inventory.InventoryView;
+ import org.bukkit.inventory.SmithingInventory;
++import org.spigotmc.SpigotConfig;
+ // CraftBukkit end
+ 
+ public class ServerGamePacketListenerImpl implements ServerPlayerConnection, TickablePacketListener, ServerGamePacketListener {
+@@ -2256,7 +2257,8 @@ public class ServerGamePacketListenerImpl implements ServerPlayerConnection, Tic
+     private void performChatCommand(ServerboundChatCommandPacket packet) {
+         // CraftBukkit start
+         String command = "/" + packet.command();
+-        ServerGamePacketListenerImpl.LOGGER.info(this.player.getScoreboardName() + " issued server command: " + command);
++        if (SpigotConfig.logCommands) // Paper - option was not respected
```

Comment:
Make sure you wrap your diff in paper comments.
Also use a fully qualified import.
Don't change the indentation of the logging message, this allows you to reduce the diff.
```java
if (org.spigotmc.SpigotConfig.logCommands) // Paper - option was not respected
ServerGamePacketListenerImpl.LOGGER.info(this.player.getScoreboardName() + " issued server command: " + command);
} // Paper
```